### DISCUSSION TITLE
Solaredge Battery Available Energy - Unknown

### DISCUSSION THREAD

**Discussion by Danifly80** [WillCodeForCats/solaredge-modbus-multi#700] [General]
### Describe the bug

Battery Available energy sensor are "unknown" but all the other sensor works without any issues.

The sensor works for some days at the beginning but after some days (see screenshot of historic data) it stating report "unknown" value.
the other sensor for the battery seams to work without problem.

thanks for your work and you help

### Expected behavior

have a valid value for the sensor

### Screenshots

![immagine](https://github.com/user-attachments/assets/048c1bb6-e2ae-4884-a379-6ad211e99f71)
![immagine](https://github.com/user-attachments/assets/a3bc4666-6727-4ae5-a2d5-22e2e0b45d83)
![immagine](https://github.com/user-attachments/assets/12ff61d0-65c4-4fe4-bf7e-3c6c543cf344)

### Diagnostic File

[home-assistant_solaredge_modbus_multi_2024-11-15T22-10-42.294Z.log](https://github.com/user-attachments/files/17782261/home-assistant_solaredge_modbus_multi_2024-11-15T22-10-42.294Z.log)

### Debug logs

```shell
the sensor in the debug report every time the same value:

DEBUG (MainThread) [custom_components.solaredge_modbus_multi.hub] I1B1: B_Energy_Available 0x461c1eb8 <class 'float'>
```

### Home Assistant Version

2024.11.2

### solaredge-modbus-multi Version

3.0.1

### Installation Type

Core

### Read the Instructions

- [X] My configuration follows https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration
- [X] I have read the Known Issues https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Known-Issues
- [X] This issue is not about a template error (do not report template errors, read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Design-Notes)

**Comment by WillCodeForCats** [comment]
Reported energy available is greater than reported maximum energy, set your battery rating adjustment.

https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration#battery-options

**Reply by Danifly80** [reply]
Hi,

Thanks for your reply and your help.

Is there a way to know the right percentage for the "Battery Rating Adjustment" ?
I put 90% and now there is a value but it is bigger than the Maximum Energy

![immagine](https://github.com/user-attachments/assets/ff4fea8b-85bf-4ceb-a9a6-6e71894e8cff)

**Reply by WillCodeForCats** [reply]
Yes that's what the setting is for.

You choose if you want to see the over-rating value or unavailable state.

I don't have batteries so I can't offer any advice which is better.
